WordPress/Classes/ViewRelated/Post/PostPreviewViewController.m

```diff
@@ -180,7 +180,7 @@ - (void)webView:(UIWebView *)webView didFailLoadWithError:(NSError *)error
 
     [self stopLoading];
 
-    [self.generator previewRequestFailedWithError:error];
+    [self.generator previewRequestFailedWithReason:@"Generic web view error"];
```

Comment:
I think it would still be valuable to log the error domain/code so we can differentiate expected errors like timeouts from things that we might want to address.